orca-keel/src/main/groovy/com/netflix/spinnaker/orca/config/KeelConfiguration.java

```diff
@@ -26,12 +26,14 @@
 import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
 import retrofit.Endpoint;
 import retrofit.Endpoints;
 import retrofit.RestAdapter;
 import retrofit.client.Client;
 import retrofit.converter.JacksonConverter;
 
+@Configuration
```

Comment:
This is what was preventing Travis from compiling.

Reply:
Arg thank you